userdocs/profiles.dev/docs/author-docs/kustomize-and-raw.md

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_position: 4
+---
+
+# Kustomize patches and raw yaml artifacts
+
+Both Kustomize patches and raw yamls (such as a simple deployment manifest,
+or any other Kubernetes object) can be added to a profile under the same key.
+
+The resources must be stored locally within the profile directory.
+For example:
+
+```bash
+org-profiles-repo/
+├── our-awesome-apps-profile
+│   ├── super-cool-artifact-manifests
+│   │   ├── deployment.yaml
+│   │   └── patches.yaml
+│   └── profile.yaml
+...
+```
+
+Then in the `profile.yaml` we add these artifact manifests by using the `kustomize`
+type identifier:
+
+```yaml
+# ...
+spec:
+  # ...
+  artifacts:
+    - name: # the name of your artifact as you would like it to be known in the profile
+      kustomize:
+        path: "super-cool-artifact-manifests/*" # the relative path to the manifests directory
```

Comment:
@aclevername I never actually tried, but do we support `*`?

Reply:
I would guess not since I *think* we do some `filepath.Join(path, value)`. I think we should remove it :smile:

Reply:
Wait what is the right thing here? I asked about this in slack and was told I didn;t need to do an artifact per file

Reply:
> Wait what is the right thing here? I asked about this in slack and was told I didn;t need to do an artifact per file


yes thats still true. Its just `path: "super-cool-artifact-manifests/"`, no need for `*`